.prettierignore

```diff
@@ -31,3 +30,6 @@ next-env.d.ts
 # native
 android
 ios
+
+# submodules
+packages/transport/trezor-common/
```

Comment:
this does not work for me, prettier is still linting submodule. is there some config for prettier anywhere else? @Nodonisko

Reply:
Hmm that will be issue because `.prettierignore` file is actually not used and I forged to remove that file.

If you check package.json prettier scripts, you will see that prettier uses .gitignore, because till this change, these two files was exactly same.

What are actual errors here? I don't see any pipelines output. If is problem with JSON, maybe just override prettier config for `json` in this submodule, to match style? Usually there is only mismatch in tabWith or something similar. Check `.prettierrc` how this is done.

Reply:
If we will need to ignore this I can think of two possible solutions:
1. Copy content of `.gitignore` to `.prettierignore` and remove use of gitignore in `package.json` prettier scripts
2. Adding submodule to gitignore - I think this should work but we wont be able actually commit changes to submodule for suite repo.

Reply:
There are no errors produced by prettier over trezor-common submodule. Maybe some errors could occur in future. 
I think we want to have submodule checked into version control.

Reply:
In that case there is only one possible solution n.1. Please copy paste content of gitignore to prettierignore and on top create some section with "prettier extra ignore" so we can easily copy paste again in future if we will do changes in gitignore

Reply:
remove `--ignore-path  .gitignore` from `package.json` scripts `format` and `format:verify` then it should fallback to prettierignore

Reply:
seems to work. nice.